cmd/docker-proxy/network_proxy_test.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 var (
-	testBuf     = []byte("Buffalo buffalo Buffalo buffalo buffalo buffalo Buffalo buffalo")
+	testBuf     = []byte("Buffalo1 buffalo2 Buffalo3 buffalo4 buffalo5 buffalo6 Buffalo7 buffalo8")
```

Comment:
😅

Reply:
LOL yeah, not very creative there 😂 I don't think the output here was anything special, so I could've "Lorum Ipsum'd" it 😂